rskj-core/src/main/java/org/ethereum/net/client/ConfigCapabilitiesImpl.java

```diff
@@ -29,39 +29,47 @@
 import java.util.TreeSet;
 
 import static org.ethereum.net.client.Capability.RSK;
+import static org.ethereum.net.client.Capability.SNAP;
+import static org.ethereum.net.client.Capability.SNAP_VERSION;
 import static org.ethereum.net.eth.EthVersion.fromCode;
 
 /**
  * Created by Anton Nashatyrev on 13.10.2015.
  */
 public class ConfigCapabilitiesImpl implements ConfigCapabilities{
 
-    private final SystemProperties config;
+    private final RskSystemProperties config;
 
-    private SortedSet<Capability> allCaps = new TreeSet<>();
+    private SortedSet<Capability> allCapabilities = new TreeSet<>();
 
-    public ConfigCapabilitiesImpl(SystemProperties config) {
+    public ConfigCapabilitiesImpl(RskSystemProperties config) {
         if (config.syncVersion() != null) {
             EthVersion eth = fromCode(config.syncVersion());
             if (eth != null) {
-                allCaps.add(new Capability(RSK, eth.getCode()));
+                allCapabilities.add(new Capability(RSK, eth.getCode()));
             }
         } else {
             for (EthVersion v : EthVersion.supported()) {
-                allCaps.add(new Capability(RSK, v.getCode()));
+                allCapabilities.add(new Capability(RSK, v.getCode()));
             }
         }
+
+        if (config.isServerSnapshotSyncEnabled() && allCapabilities.stream().anyMatch(Capability::isRSK)) {
+            allCapabilities.add(new Capability(SNAP, (byte) SNAP_VERSION));
```

Comment:
suggestion: 

Change the constant `SNAP_VERSION` to byte. It's just used here, it's 1, I don't see advantage create it as int to be casted after.

Reply:
solved!